src/main/scala/com/actian/spark_vector/vector/ColumnMetadata.scala

```diff
@@ -58,5 +60,6 @@ case class ColumnMetadata(val name: String,
   def maxDataSize: Int = dataTypeInfo._2
 
   override def toString: String =
-    s"name: ${name}, typeName: ${typeName}, dataType: ${dataType}, nullable: ${nullable}, precision: ${precision}, scale: ${scale}"
+    s"name: ${name}, typeName: ${typeName}, dataType: ${dataType}, nullable: ${nullable}, precision: ${precision}, scale: ${scale}" +
```

Comment:
put a `, ` (comma and space) at the end of this so that the `value_ranges` be formatted correctly